src/career.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Career</title>
+  </head>
+  <body>
+    <h1>The Career of Keanu Reeves</h1>
+    <!-- Intro with basic info on Reeves -->
+    <h2>Intro to Keanu Reeves</h2>
+    <p>
+      Keanu Reeves is a Canadian Movie star that is famous for his many
+      blockbuster roles. Staring in the Bill and Ted franchise, the Matrix
+      franchise and the John Wick movies. To name a few. Reeves also plays the
+      bass guitar for the bands Dogstar and Becky. He has also Acted onstage, he
+      performed as Prince Hamlet for the Manitoba Theatre Centre's production of
+      Hamlet. He has also directed one movie Man of Chi.
+      <!-- List of Skill and talents -->
+    </p>
+
+    <h2>Skills and Talents</h2>
+    <ul>
+      <li>Acting of course</li>
+      <li>Playing base guitar</li>
+      <li>Knowing different fighting styles and martial arts</li>
+    </ul>
+    <!-- List of different movies -->
+    <h2>List of Movies</h2>
+    <ul>
+      <div>
+        <img
+          src="img/John wick 3.jpeg"
+          alt="John Wick 3 poster"
+          width="400"
+          height="500"
+        />
+      </div>
+      <li>John Wick: Chapter 3 - Parabellum</li>
+      <div>
+        <img src="img/Bill and Ted.jpg" alt="Bill and Ted Poster" width="400" />
+      </div>
+      <li>Bill and Ted</li>
+      <div>
+        <img src="img/Constantine poster.jpg" alt="Constantine Poster" />
+      </div>
+      <li>Constantine</li>
+      <div>
+        <img
+          src="img/Devils Advocate.jpg"
+          alt="Devil Advocate Poster"
+          width="400"
+        />
+      </div>
+      <li>The Devil's Advocate</li>
+    </ul>
```

Comment:
While I don't think it's anything other than a semantic oddity, I don't think I've ever seen things other than <li> nested inside <ul> tags. There may be a better way to handle this, but I think for our immediate purposes, you're good. Just maybe something to look into in the future.

Reply:
You are right I've never seen an image within a list. But the list items correspond with the images that's why I nested them within the list. If the images being in the list have any effect on the style I will move the images out of the list.  👍